packages/flutter/lib/src/widgets/service_extensions.dart

```diff
@@ -376,6 +392,8 @@ enum WidgetInspectorServiceExtensions {
   ///   encoded String representation of this data.
   /// * [WidgetInspectorService.initServiceExtensions], where the service
   ///   extension is registered.
+  ///
+  /// !!IMPORTANT!! Please use [getRootWidgetTree] instead.
```

Comment:
Added these comments because I'm not sure we should (or even can?) deprecate service extensions.

Reply:
I think you can annotate with deprecated and say use getRootWidgetTree instead

Reply:
Ah nice, done!